qcog_python_client/qcog.py

```diff
@@ -137,14 +146,16 @@ def ensemble(
         target_operator: list[Operator] = [],
     ) -> Any:
         """Select EnsembleModel for the training."""
-        self.model = EnsembleModel(
-            operators,
-            dim,
-            num_axes,
-            sigma_sq,
-            sigma_sq_optimization,
-            seed,
-            target_operator,
+        # Cast all the operators to string
+        self.model = ModelEnsembleParameters(
+            operators=[str(op) for op in operators],
+            dim=dim,
+            num_axes=num_axes,
+            sigma_sq=sigma_sq,
+            sigma_sq_optimization_kwargs=sigma_sq_optimization,
+            seed=seed,
+            target_operators=[str(op) for op in target_operator],
+            model_name="ensemble",
```

Comment:
This seems to be actually a glitch of the code generation, because `ensemble` is the only parameter allowed for the ensemble model and it's a literal, so I should not be able to specify it. I will look if there is anything in the generated code that can go here instead of the literal, but this is validated at runtime anyway and it would fail linting if anything different than that is provided

Reply:
You are 100% right, Model is auto generated so it fits perfectly